flang/test/Lower/intrinsic-procedures/command_argument_count.f90

```diff
@@ -0,0 +1,10 @@
+! RUN: bbc -emit-fir %s -o - | FileCheck %s
+
+! CHECK-LABEL: argument_count_test
+subroutine argument_count_test()
+integer :: arg_count_test
+arg_count_test = command_argument_count()
+! CHECK: %[[i0:.*]] = fir.alloca i32 {bindc_name = "arg_count_test", uniq_name = "_QFargument_count_testEarg_count_test"}
```

Comment:
Minor nit: I think you can skip the alloca attributes, they're not directly relevant to the test and they might change in the future.

Reply:
Will do. Do you think its worth removing: `! CHECK: fir.store %[[i1]] to %[[i0]] : !fir.ref<i32>` as well, since the test only really cares about the intrinsic being called?

Reply:
Done. Also removed the fir.store as well since its not directly relevant.

Reply:
Right, seems fine now, thanks :)